code/modules/antagonists/eldritch_cult/knowledge.dm/flesh_lore.dm

```diff
@@ -0,0 +1,257 @@
+	if(H.stat != DEAD)
+		to_chat(user, "<span class='warning'>This spell can only affect the dead!</span>")
+		return
+
+	for(var/mob/dead/observer/ghost in GLOB.dead_mob_list) //excludes new players
+		if(ghost.mind && ghost.mind.current == H && ghost.client)  //the dead mobs list can contain clientless mobs
+			ghost.reenter_corpse()
+			break
+	//Remember to un comment it when it gets TMed
+	//if(!H.mind || !H.client)
+	//	to_chat(user, "<span class='warning'>There is no soul connected to this body...</span>")
+	//	return
+
+	check_ghouls()
+
+	if(LAZYLEN(spooky_scaries) >= ghoul_amt)
+		to_chat(user, "<span class='warning'>Your patron cannot support more ghouls on this plane!</span>")
+		return
+
+	LAZYADD(spooky_scaries,H)
+
+	ADD_TRAIT(H,TRAIT_GHOUL,MAGIC_TRAIT)
+	H.revive(full_heal = TRUE, admin_revive = TRUE)
+	H.setMaxHealth(25)
+	H.health = 25
+
+	H.fully_replace_character_name(H.real_name,"Ghouled [H.real_name]")
+	to_chat(H, "<span class='userdanger'>You have been revived by </span><B>[user.real_name]!</B>")
+	to_chat(H, "<span class='userdanger'>[user.p_theyre(TRUE)] your master now, assist [user.p_them()] even if it costs you your new life!</span>")
+	return
+
+/datum/eldritch_knowledge/flesh_grasp/proc/check_ghouls()
+	if(LAZYLEN(spooky_scaries) == 0)
+		return
+
+	for(var/X in spooky_scaries)
+		if(!ishuman(X))
+			LAZYREMOVE(spooky_scaries,X)
+			continue
+		var/mob/living/carbon/human/H = X
+		if(H.stat == DEAD)
+			H.dust(TRUE)
+			LAZYREMOVE(spooky_scaries,X)
+			continue
+
+	listclearnulls(spooky_scaries)
+
+/datum/eldritch_knowledge/flesh_mark
+	name = "Mark of flesh"
+	gain_text = "I saw them, the marked ones. The screams.. the silence."
+	desc = "Your sickly blade now applies mark of flesh status effect. To proc the mark, use your mansus grasp on the marked. Mark of flesh when procced causeds additional bleeding."
+	cost = 2
+	next_knowledge = list(/datum/eldritch_knowledge/summon/raw_prophet)
+	banned_knowledge = list(/datum/eldritch_knowledge/rust_mark,/datum/eldritch_knowledge/ash_mark)
+	route = "Flesh"
+
+/datum/eldritch_knowledge/flesh_mark/eldritch_blade_act(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(istype(target,/mob/living))
+		var/mob/living/L = target
+		L.apply_status_effect(/datum/status_effect/eldritch/flesh)
+
+/datum/eldritch_knowledge/flesh_blade_upgrade
+	name = "Bleeding Steel"
+	gain_text = "It rained blood, that's when i understood the gravekeeper's advice."
+	desc = "Your blade will now cause additional bleeding."
+	cost = 2
+	next_knowledge = list(/datum/eldritch_knowledge/summon/stalker)
+	banned_knowledge = list(/datum/eldritch_knowledge/ash_blade_upgrade,/datum/eldritch_knowledge/rust_blade_upgrade)
+	route = "Flesh"
+
+/datum/eldritch_knowledge/flesh_blade_upgrade/eldritch_blade_act(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/C = target
+		C.bleed_rate++
```

Comment:
holy shit this is so pathetic

Reply:
yeah i uhh. gotta find a good replacment for these